content/en/blog/2019/app-identity-and-access-adapter/index.md

```diff
@@ -0,0 +1,167 @@
+---
+title: App Identity and Access Adapter
+subtitle: Using Istio to Secure Multicloud Kubernetes Applications with Zero Code Changes
+description: Using Istio to Secure Multicloud Kubernetes Applications with Zero Code Changes.
+publishdate: 2019-09-01
+attribution: Anton Aleksandrov (IBM)
+keywords: [security,oidc,jwt,policies]
+---
+
+Whether your computing environment uses multiple cloud providers, a single cluster, a combination of on- and off-premise solutions, or multiple containers in one cloud, having a centralized identity management can help you to preserve existing infrastructure and avoid vendor lock-in.
+
+Simply put, if you are running your containerized applications on Kubernetes, you can benefit from using the App Identity and Access Adapter for an abstracted level of security with zero code changes and no redeployments.
+
+With the [App Identity and Access Adapter](https://github.com/ibm-cloud-security/app-identity-and-access-adapter), you can use any OAuth2/OIDC provider, such as IBM Cloud App ID, Auth0, Okta, Ping Identity, AWS Cognito, Azure AD B2C and more. Authentication and control authorization policies can be applied in a streamlined way in all environments — including frontend and backend applications — without any changes to your code or the need to redeploy your application.
+
+## Understanding Istio and the adapter
+
+[Istio](/docs/concepts/what-is-istio/) is an open source service mesh that layers transparently onto existing distributed applications that can integrate with Kubernetes. To reduce the complexity of deployments Istio provides behavioral insights and operational control over the service mesh as a whole. [See Istio Architecture for more details.](/docs/concepts/what-is-istio/#architecture)
+
+Istio uses a sidecar model with Envoy proxy to mediate all inbound and outbound traffic for all pods in the service mesh. By using the Envoy, Istio extracts information about network traffic, also known as telemetry, that is sent to the Istio component called [Mixer](/docs/concepts/what-is-istio/#mixer), which is responsible for collecting telemetry and enforcing policy decisions.
```

Comment:
* "Istio uses *the* sidecar model" or "Istio uses Envoy proxy sidecars to mediate..."
* For the second sentence, perhaps something like "Istio extracts telemetry from the Envoy sidecars and sends it to Mixer, the Istio component responsible for collecting telemetry and policy enforcement".

Reply:
You might link the first mention of the side car model to another doc, like this one:
https://istio.io/blog/2019/data-plane-setup/

Sidecar models are likely familiar to developers but this new wave of IT Pro users are new to it and it's an important concept to understand the value of Istio in general.